ui/src/Components/ImageLoad.jsx

```diff
@@ -30,7 +30,7 @@ function ImageLoad(props) {
     setTryAgain(false);
     setTimeoutID();
 
-    const src = new RegExp("/^(?:/|[a-z]+://)/").test(props.src) ? props.src : `/${props.src}`;
+    const src = new RegExp("^(?:[a-z]+:)?//").test(props.src) ? props.src : `/${props.src}`;
```

Comment:
whats this modification for?

Reply:
The previous one wasn't detecting an absolute URL properly (e.g. tmdb search result images)